src/app/authorization/registration/registration.component.ts

```diff
@@ -122,45 +95,42 @@ export class RegistrationComponent {
       this.registrationForm.value.mobile
     );
   }
-  Mustmatch(pwd: any, cpwd: any) {
-    debugger;
-    return () => {
-      debugger;
-      const passwordcontrol = this.registrationForm.controls[pwd];
-      const confirmpasswordcontrol = this.registrationForm.controls[cpwd];
-      if (
-        confirmpasswordcontrol.errors &&
-        !confirmpasswordcontrol.errors['Mustmatch']
-      ) {
-        return;
-      }
-      if (passwordcontrol.value !== confirmpasswordcontrol.value) {
-        confirmpasswordcontrol.setErrors({ Mustmatch: true });
-      } else {
-        confirmpasswordcontrol.setErrors(null);
-      }
-    };
-  }
 
-  get FullName(): FormControl {
-    return this.registrationForm.get('fullname') as FormControl;
-  }
-  get Pwd(): FormControl {
-    return this.registrationForm.get('pwd') as FormControl;
-  }
-  get Cpwd(): FormControl {
-    return this.registrationForm.get('cpwd') as FormControl;
-  }
-  get Gender(): FormControl {
-    return this.registrationForm.get('gender') as FormControl;
-  }
-  get Birthday(): FormControl {
-    return this.registrationForm.get('birthday') as FormControl;
+  createForm() {
+    this.registrationForm = this.fb.group({
+      id: new FormControl(Date.now()),
```

Comment:
Why FormControl is used in form Builder ?

Reply:
yes, I have used in form builder and done this changes.